--- a/data/meson.build
+++ b/data/meson.build
@@ -21,15 +21,3 @@
   [ 'its/shared-mime-info.loc', 'its/shared-mime-info.its', ],
   install_dir : get_option('datadir') / 'gettext/its'
 )
-
-custom_target('shared-mime-info-spec-html',
-    input : 'shared-mime-info-spec.xml',
-    output: 'shared-mime-info-spec-html',
-    command: [
-        xmlto,
-        '-o', '@OUTPUT@',
-        'html-nochunks',
-        '@INPUT@',
-    ],
-    build_by_default: true,
-)
--- a/meson.build
+++ b/meson.build
@@ -22,7 +22,6 @@
 
 itstool = find_program('itstool')
 xmllint = find_program('xmllint')
-xmlto   = find_program('xmlto')
 
 ###############################################################################
 # Find xdgmime
