From fa09e9697e7e6ac09b58d374e7a6eff6b44e26dd Mon Sep 17 00:00:00 2001
From: SupervisedThinking <supervisedthinking@gmail.com>
Date: Sat, 6 Mar 2021 21:39:19 +0100
Subject: [PATCH] Audiomanager: adjust music folder paths

---
 es-core/src/AudioManager.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/es-core/src/AudioManager.cpp b/es-core/src/AudioManager.cpp
index e0fef1e4d..e81e03ba9 100644
--- a/es-core/src/AudioManager.cpp
+++ b/es-core/src/AudioManager.cpp
@@ -152,13 +152,13 @@ void AudioManager::playRandomMusic(bool continueIfPlaying)
 
 	std::vector<std::string> musics;
 	
-	// check in RetroPie music directory
+	// check in storage music directory
 	if(musics.empty())
-		getMusicIn(Utils::FileSystem::getHomePath() + "/RetroPie/roms/music", musics);
+		getMusicIn("/storage/music/es", musics);
   
 	// check in system sound directory
 	if(musics.empty())
-		getMusicIn("/usr/share/RetroPie/music", musics);
+		getMusicIn("/usr/share/emulationstation/music", musics);
   
 	// check in .emulationstation/music directory
 	if(musics.empty())
