--- a/configure.ac
+++ b/configure.ac
@@ -466,9 +466,9 @@
    AC_MSG_RESULT(no)
 fi
 
-AC_CHECK_LIB(GLEW, glewInit, , AC_MSG_ERROR([GLEW not found]),[])
+AC_CHECK_LIB(glew, glewInit, , AC_MSG_ERROR([GLEW not found]),[])
 AC_CHECK_HEADER([GL/glew.h],[],[AC_MSG_ERROR([You need to install the development files for GLEW. On Debian-style distros this is libglew-dev])])
-LIBS="$LIBS -lGLEW"
+LIBS="$LIBS -lglew"
 
 AH_TEMPLATE(C_FLUIDSYNTH,[Define to 1 to enable fluidsynth MIDI synthesis])
 AC_CHECK_HEADER(fluidsynth.h,have_fluidsynth_h=yes,)

