From 761fe1a6eb39c2cbb0a789d1b58692b293f63380 Mon Sep 17 00:00:00 2001
From: SupervisedThinking <supervisedthinking@gmail.com>
Date: Sun, 28 Feb 2021 21:34:12 +0100
Subject: [PATCH] Makefile: fix cross compile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4872239e..f66d5e57 100644
--- a/Makefile
+++ b/Makefile
@@ -211,7 +211,7 @@ $(error Unknown platform:$(PLATFORM))
 endif
 
 RM     = rm -f
-AS     = as
+AS     ?= as
 CC     ?= gcc
 CXX    ?= g++
 STRIP  ?= strip
